src/SFA.DAS.ApprenticeCommitments.Jobs.Functions/ApprenticeshipCommitmentsJobsHandler.cs

```diff
@@ -43,22 +48,19 @@ public async Task Handle(ApprenticeshipCreatedEvent message, IMessageHandlerCont
                 var res = await _api.CreateApprentice(message.ToApprenticeshipCreated());
 
                 _logger.LogInformation($"CreateApprentice returned {JsonConvert.SerializeObject(res)}");
-                _logger.LogInformation($"_nServiceBusOptions {JsonConvert.SerializeObject(_nServiceBusOptions)}");
+                _logger.LogInformation($"_nServiceBusOptions {JsonConvert.SerializeObject(_settings)}");
```

Comment:
The line above this 
_logger.LogInformation($"CreateApprentice returned {JsonConvert.SerializeObject(res)}");

should never be logged, I thought it had been removed (as I'm sure we aren't supposed to log PII data)